src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-list/cephfs-subvolume-list.component.ts

```diff
@@ -147,16 +146,17 @@ export class CephfsSubvolumeListComponent extends CdForm implements OnInit, OnCh
       }
     ];
 
-    this.subVolumes$ = this.subject.pipe(
+    this.getSubVolumes();
+
+    this.subVolumeGroups$ = this.subject.pipe(
```

Comment:
a different subject should track the subvolume group.
```suggestion
    this.subVolumeGroups$ = this.groupSubject.pipe(
```